recipes/git-subrepo/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "git-subrepo" %}
+{% set version = "0.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ingydotnet/git-subrepo/archive/{{ version }}.tar.gz
+  sha256: d2e3cc58f8ac3d90f6f351ae2f9cc999b133b8581ab7a0f7db4933dec8e62c2a
+
+build:
+  number: 0
+  script: make install
```

Comment:
Is there a `make check` that we can run?

Reply:
The `Makefile` is actually only copying sources into the prefix, so there's no real build being execute for `make check`.